runtime_lib/xaiengine/CMakeLists.txt

```diff
@@ -8,10 +8,11 @@ cmake_minimum_required(VERSION 3.20.1)
 
 project("xaiengine lib for ${AIE_RUNTIME_TARGET}")
 
-set(xaienginePath ${VITIS_ROOT}/data/embeddedsw/XilinxProcessorIPLib/drivers/aienginev2_v3_0/src) 
-#set(xaienginePath ${VITIS_AIETOOLS_DIR}/include/drivers/aiengine)
-file(GLOB libheaders ${xaienginePath}/*.h)
-file(GLOB libheadersSub ${xaienginePath}/*/*.h)
+set(xaienginePath ${VITIS_AIETOOLS_DIR}/include/drivers/aiengine)
+#tmp path to header files since we use an older (3_0) version of xaiengine. need to fix when we upgrade to 2023.1
+set(xaienginePathTmp ${VITIS_ROOT}/data/embeddedsw/XilinxProcessorIPLib/drivers/aienginev2_v3_0/src) 
+file(GLOB libheaders ${xaienginePathTmp}/*.h ${xaienginePath}/xioutils.h)
+file(GLOB libheadersSub ${xaienginePathTmp}/*/*.h)
 
```

Comment:
I don't understand this change.  Seems like it does nothing?  or it's just picking up xioutils.h?

Reply:
@stephenneuendorffer yes, we need to pick up  xioutils.h. This is a temp work around since we need to pick up older header files corresponding to the Vitis version that was used to generate the images for our boards (xaienginev2 v3_0). This will be cleaned up after we upgrade fully to 2023.1 once it is released.